EIPS/eip-7756.md

```diff
@@ -0,0 +1,177 @@
+---
+eip: 7756
+title: EOF/EVM Trace Specification
+description: Updates EIP-3155 tracing to add EOF support
+author: Martin Holst Swende (@holiman), Marius van der Wijden (@MariusVanDerWijden), Danno Ferrin (@shemnon)
+discussions-to: https://ethereum-magicians.org/t/eip-7756-eof-evm-trace-specification/20806
+status: Draft
+type: Standards Track
+category: Interface
+created: 2024-08-13
+requires: 3155
+---
+
+## Abstract
+
+Updates the [EIP-3155](./eip-3155.md) JSON tracing specification to support EOF features and light tracing.
```

Comment:
Would be nice to see a definition of "light tracing" in the abstract. It isn't a term I'm familiar with.

Reply:
Half baked thought, deleted.  

Light tracing deserves anther spec, but would be focused on differential fuzzing and hence would only have essential fields with terse names or an SSZ/RLP encoding.  Gas remaining, depth, top of stack, pc, section number, memory size.  I think that's the essential items needed for differential fuzzing.  That's something I expect would take the prototype then standardize route.  @guidovranken's tool is based on protobuff, for example.